lib/vagrant-bhyve/driver.rb

```diff
@@ -146,6 +146,28 @@ def load_module(module_name)
 	end
       end
 
+      def check_or_create_default_pfconf(ui)
+        if execute(true, "test -s /etc/pf.conf") != 0
+	  ui.warn I18n.t("vagrant_bhyve.action.vm.boot.create_default_pfconf")
+
+          # probably this could be done in a nicer way with open and puts...
```

Comment:
I'll replace this with a `File.open(...)` block when I'm rebasing the next time.